cellarium/ml/module/base_module.py

```diff
@@ -39,13 +41,7 @@ class PredictMixin(metaclass=ABCMeta):
     """
 
     @abstractmethod
-    def predict(self, x_ng: torch.Tensor) -> torch.Tensor:
+    def predict(self, *args: Any, **kwargs: Any) -> torch.Tensor | ModelOutput:
```

Comment:
This is too broad of a mixin to be practically meaningful.  I suggest generalizing the original signature to the degree that might be needed, e.g. perhaps taking `Dict[str, torch.Tensor]` and input and `Dict[str, torch.Tensor]` as output, and wrapping huggingface models that produce `ModelOutput` to conform to _our_ predict mixin.